scripts/update_golden.sh

```diff
@@ -46,13 +46,4 @@ else
     ARG=""
 fi
 
-if [[ $DOCKER == "true" ]]; then
-  DOCKER_BUILDKIT=1 docker build --tag datacommons/mixer-golden-update  -f build/Dockerfile --target golden-update .
```

Comment:
could you also document this removal in the PR description?

Reply:
Done. The docker mode is never used, so removed to skip updating it.